apps/app/src/views/routes/NewOrder/components/SelectPharmacyCard/index.tsx

```diff
@@ -12,14 +12,16 @@ import {
   useDisclosure
 } from '@chakra-ui/react';
 
-import { types } from '@photonhealth/react';
+import { types, usePhoton } from '@photonhealth/react';
 
 import { LocationSearch } from '../../../../components/LocationSearch';
 import { LocalPickup } from './components/LocalPickup';
 import { MailOrder } from './components/MailOrder';
 import { Address } from '../../../../../models/general';
 import { SendToPatient } from './components/SendToPatient';
 
+const pharmacySelectionEnabledOrgs = [process.env.REACT_APP_MODERN_PEDIATRICS_ORG_ID];
```

Comment:
Like this. Think we can do something similar with the `onlyCurexa` field? Not sure if that'll make it more confusing or not.

Maybe the inverse?